.github/workflows/publish.yml

```
@@ -31,8 +31,7 @@ concurrency:
 jobs:
   push-images-to-production-artifacts:
     if:
-      ${{ github.ref == 'refs/heads/main' && (github.event_name == 'release' ||
-      inputs.foolproof == true ) }}
+      ${{ github.ref == 'refs/heads/main' || github.event_name == 'release' || inputs.foolproof == true }}
```

We only allow running manual release actions from `main`:
```suggestion
      ${{ github.event_name == 'release' || (github.ref == 'refs/heads/main' && inputs.foolproof == true) }}
```